src/assets/ui/menu_bg.webp.import

```diff
@@ -3,25 +3,26 @@
 importer="texture"
 type="CompressedTexture2D"
 uid="uid://dwu02f18und58"
-path="res://.godot/imported/menu_bg.webp-4b99d77d83c98476cbd10cc996179e34.ctex"
+path.s3tc="res://.godot/imported/menu_bg.webp-4b99d77d83c98476cbd10cc996179e34.s3tc.ctex"
 metadata={
-"vram_texture": false
+"imported_formats": ["s3tc_bptc"],
+"vram_texture": true
 }
 
 [deps]
 
 source_file="res://assets/ui/menu_bg.webp"
-dest_files=["res://.godot/imported/menu_bg.webp-4b99d77d83c98476cbd10cc996179e34.ctex"]
+dest_files=["res://.godot/imported/menu_bg.webp-4b99d77d83c98476cbd10cc996179e34.s3tc.ctex"]
 
 [params]
 
-compress/mode=0
+compress/mode=2
```

Comment:
this changes it from Lossless to VRAM compressed, which technically isn't recommended for purely 2D art but after checking in the engine, the quality seems the same and it saves us some VRAM in exchange for PCK size, so I think it's okay. Given the title of the PR I wasn't sure if you meant to change it 😂 but I'm happy with it